static/src/stylesheets/module/facia/_item.scss

```diff
@@ -188,14 +188,19 @@ $fc-item-gutter: $gs-gutter / 4;
     }
 
     .u-faux-block-link--hover {
-        background-color: colour(neutral-6);
+        // For touch devices we want to reduce the time before the click event
+        // as much as possible, because browsers on touch devices give feedback
+        // at the end of the click event.
```

Comment:
This comment is fine, but can it just be tweaked to explain how background-color affects touch event response time?